packages/rn-tester-e2e/scripts/run-e2e-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
given that this is a new script. Can we have it in JS instead of in bash?

This will make it more portable and JS is easier to read and reason about. Also, we have linters and formatters for it.

Reply:
Also... can we create a folder into `scripts/e2e` where we put these scripts? This will keep the folder more organized.

Reply:
good idea, changed.

Reply:
> Also... can we create a folder into scripts/e2e where we put these scripts? This will keep the folder more organized.

Would you like me to move all scripts related to running e2e to this folder?